src/PresentationalComponents/Inventory/Inventory.js

```diff
@@ -113,90 +118,32 @@ const Inventory = ({
         selected: selectedIds,
       },
     });
-    checkRemediationButtonStatus();
```

Comment:
Maybe this should/could be a hook, something like `useRemediationButtonStatus`, that includes the  `isRemediationButtonDisabled` state and returns it. Maybe the same can be done with the `pathwayCheck` below.

Reply:
For pathway/rule Check, since they only run once on page load, what would be the benefit of changing those to hooks?